MIGRATION.md

```diff
@@ -0,0 +1,88 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (package: `@cloudant/cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`) that is JavaScript and TypeScript compliant.
```

Comment:
Out of curiosity, why did you change compatible with compliant? Compatible seems to be the better word for the relationship between `cloudant-node-sdk` and JS/TS.

Reply:
Okay, then I revert it back :)

Reply:
Changed in 40dd63a